### DISCUSSION TITLE
Seawolf/Yasen and Tier 10 Submarine Discussion

### DISCUSSION THREAD

**Discussion by Therightroad2009** [SoftbearStudios/mk48#209] [Ideas]
Hello everyone. Hope you are doing good :).. I want to talk about SW and Yasen in terms of stats, armament and sensors. AND ALSO TIER 10 IDEAS

heres all of the text:
**ARMAMENT** 
- I want to start talking about the armament yasen has: The SET-65 torpedoes are too old, for being a 2013 submarine adding a SET-65 to it is not something i would agree with.
The way to fix this is adding the Futlyar torpedo (also named Fizik-2). Basically it's the newest russian torpedo which new russian submarines use.
**STATS OF FUTLYAR:**
Length: 7.2m
Speed: 60 knots
Lifespan: 75-80 seconds
Reload: 12 seconds
Damage: 1.20
- Seawolf countermeasures and reload: Make seawolf have 4 decoys with a 15 sec reload (Maybe creating a new MOSS made certainly for Seawolf so others sub moss wont be affected)
- Make the stealth of seawolf speed 20 kts, it will be a small but useful fix.
- Add 4 FIM-92 stinger to Seawolf to defend from missiles (only use when in surface)
**STATS**
 - I don't understand how the Max Depth works in the game. But i'd recommend changing the 450+ value to 600+ Same thing for SeaWolf, 400 to 600+.
 - Add 25-35% Stealth to Yasen.
 - Increase Seawolf Stealth to 70%
**SENSORS**
Do each submarine in the game have a certain detection range for their sonar? If the answer is no, then this can't be done. If yes, then make Seawolf sonar better.
**TIER 10 SUBMARINE IDEAS:**
**SLBM:**
--------------
Vanguard class (UNITED KINGDOM)
Speed: 25kn
4 Spearfishs
1 SLBM
2 Decoys
2 SAM 
--------------
Columbia class  (UNITED STATES)
Speed: over 25kn
Stealth: 70% (same thing to seawolf)
4 MK 48s
1 SLBM
2 Decoys
2 SAM
--------------
Type 094 (CHINA)
Speed: 20kn
6 YU-6
1 SLBM
2 Decoys
2 SAM
--------------
**HUNTERS:**
--------------
Astute class (UNITED KINGDOM)
Speed: 30kn
Stealth: 50% (pump jet plus modern submarine)
6 Spearfishs
6 Decoys
4 SAM
--------------
Virginia class (UNITED STATES)
Speed: 28kn
Stealth: 70% (just same to seawolf)
4 MK 48s
12 Tomahawks
6 Decoys
4 SAM
--------------
Type 212 (ITALY AND GERMANY)
Speed: 20kn
Stealth: 20kn
4 DM2A4
2 IDAS (This IDAS can shot down helicopters/planes or missiles without being in the surface)
6 Decoys
4 SAM
--------------
Trafalgar (UNITED KINGDOM)
Speed: 30kn
Stealth: 20
5 Tigerfish
6 Decoys
4 SAM
--------------
Rubis (FRANCE)
Stealth: 0
Speed: over 25kn
4 F17 
6 Decoys
4 SAM
--------------
Suffren (FRANCE)
Stealth: 50%
Speed: over 25kn
4 F21
6 Decoys
4 SAM
--------------
Most of these don't fit good for being tier 10 because they have 4 torpedo tubes and not 6 or 8, i recommend adding 3 submarines for Tier 10: 
VIRGINIA (nuclear hunter killer submarine and also highly buffed ohio)
VANGUARD (best slbm choice) 
TYPE 212 (diesel-electric hunter killer submarine)
Sadly you can't add Borei, Typhoon, Arcturus, Oscar-II, Delta-IV, Lada, Kilo, Sierra or Victors because of the conflict :(

**Comment by finnbear** [comment]
Thanks for all the feedback and suggestions!

> The SET-65 torpedoes are too old

Maybe so, and FWIW, we also plan to add the more modern Mark 48 ADCAP torpedo for higher level USN subs.

> Make seawolf have 4 decoys with a 15 sec reload

We plan to eventually add noisemakers (they repel enemy torpedoes instead of luring them away like decoys) to the Seawolf and other submarines. This will reduce the need for tons of decoys. Note that the Seawolf launches the MK70 MOSS decoy from its torpedo tubes, reducing weapon capacity (in the game, we bend reality and allow 2 decoys and 8 torpedoes, but 4 decoys and 8 torpedoes would push that a bit far).

> Add 4 FIM-92 stinger to Seawolf to defend from missiles (only use when in surface)

I need a citation for that; we don't add weapons that don't exist in real life :wink:

> Make the stealth of seawolf speed 20 kts

It's a decent idea, at least at max depth. I think I've seen a citation for this before.

> I don't understand how the Max Depth works in the game. But i'd recommend changing the 450+ value to 600+ Same thing for SeaWolf, 400 to 600+.

1. I would need a citation
2. But, more importantly, the depth is also limited by the sea floor

> Add 25-35% Stealth to Yasen.

Is there a reason why? Some citation?

> Increase Seawolf Stealth to 70%

The Seawolf's stealth value is somewhat arbitrary, so we could do this, but I want to have a system for tracking the performance and favoritism of each ship (could be as simple as the total minutes played for each ship) before making such balancing changes.

> Do each submarine in the game have a certain detection range for their sonar?

Yes, ship sensors have varying ranges.

> make Seawolf sonar better

Is there some technical reason why it should be better relative to its size?

>  TIER 10 SUBMARINE IDEAS:

Thank you! Many of them are good candidates.

> i recommend adding 3 submarines for Tier 10

We do plan to add one, but not necessarily 3 :joy:

**Reply by Therightroad2009** [reply]
> Add 4 FIM-92 stinger to Seawolf to defend from missiles (only use when in surface)
> Your Answer:: need a citation for that; we don't add weapons that don't exist in real life 😉

Submarines have shoulder launched missiles for killing helos/planes in surface, i got this info from Dangerous Waters, a naval sim game that many actual USN people helped in, so my idea has logic.

EDIT:  i figured out usn doesnt use sams, sorry for ignorance lol

> I don't understand how the Max Depth works in the game. But i'd recommend changing the 450+ value to 600+ Same thing for SeaWolf, 400 to 600+.
> Your Answer: 
>  I would need a citation
>  But, more importantly, the depth is also limited by the sea floor

""Sources claim"" Seawolf has a max operating depth of 600m, same for Yasen. But that information is classified and no one will give exact number, so we don't know if it's 600 or more. A minimum value of 650m is ok.

> Add 25-35% Stealth to Yasen.
> Your Answer: Is there a reason why? Some citation?

The main reason i say this is Yasen is a modern submarine that has stealth. (Doesn't have a pump jet) and is bigger, which makes it to not be stealthy as Seawolf. But that doesn't mean it can't be atleast some stealthy. 30% is good

> make Seawolf sonar better
> Your Answer: Is there some technical reason why it should be better relative to its size?

I confused AN/BSY-2 Seawolf's sonar by Virginia's AN/BQQ-10(V4), Virginia has no hydrophones, it detects by light and is more sensitive.
Maybe this doesn't make sense to what we are talking about, but Seawolf is "more secret" than Virginia...
Meaning we don't know if Seawolf has same Virginia technology and it should be improved.
Can you send stats of Seawolf and Yasen please?

> I recommend adding 3 submarines for Tier 10
> Your Answer: We do plan to add one, but not necessarily 3 😂

If you add one hunter-killer submarine, we can't have a SLBM one, if you add a SLBM one, we can't have a hunter-killer sub. So what about 2? Vanguard and Virginia

> TIER 10 SUBMARINE IDEAS:
Alr thanks :)

Greets
